src/components/layout/Header.js

```diff
@@ -1,23 +1,22 @@
 import React, { useEffect, useState } from "react";
+import CurrencyExchange from "../../shared/services/currencyExchange";
+
 
 export default function Header() {
   const [data, setData] = useState([]);
```

Comment:
data, setData aren't good names. Please try to avoid generic names like, "data", "element", "item". Those don't give any information to the person that reads code